wait/exec.go

```diff
@@ -45,6 +45,13 @@ func (ws *ExecStrategy) WithStartupTimeout(startupTimeout time.Duration) *ExecSt
 	return ws
 }
 
+func (ws *ExecStrategy) WithExitCode(exitCode int) *ExecStrategy {
+	
```

Comment:
Lint will probably complain
```suggestion
```

Reply:
Fixed. Is there make command or pre-commit we should be running that will do what CI is doing?

Reply:
There is no pre-commit (https://pre-commit.com/) at the developer side, but the CI will use golangci-lint. You can run it with `make lint` from the root dir of the project.